IPFS_ALL_HANDS_FORMAT.md

```diff
@@ -0,0 +1,66 @@
+### IPFS All Hands Format
+
+![](https://media.giphy.com/media/JFawGLFMCJNDi/giphy.gif)
```

Comment:
This gif doesn't make sense to me as the gif you want people to see when they are trying to learn "the IPFS All Hands Format"

Reply:
https://i.imgur.com/mVQeLlo.gif ?